openverse_catalog/dags/util/test_pg_cleaner.py

```diff
@@ -38,1214 +42,1214 @@ def postgres_with_image_table():
-            license_="cc0",
-            license_version="1.0",
-            foreign_identifier="ark:/65665/m3272d5bfa5716461fbf173e083887621c",
-            width=None,
-            height=None,
-            creator="Susan Gabriella Stokes",
-            creator_url=None,
-            title="Eriogonum latifolium Sm.",
-            meta_data={
-                "unit_code": "NMNHBOTANY",
-                "data_source": "NMNH - Botany Dept.",
-                "license_url": "https://creativecommons.org/publicdomain/zero/1.0/",
-                "raw_license_url": "https://creativecommons.org/publicdomain/zero/1.0/",
-            },
-            raw_tags=[
-                {"name": "1930s", "provider": "smithsonian"},
-                {"name": "Dicotyledonae", "provider": "smithsonian"},
-                {"name": "United States", "provider": "smithsonian"},
-                {"name": "California", "provider": "smithsonian"},
-                {"name": "North America", "provider": "smithsonian"},
-            ],
-            watermarked=False,
-            source="smithsonian_national_museum_of_natural_history",
-        )
-    ]
-    image_store_dict = pg_cleaner.ImageStoreDict()
-    image_store_dict[("samsonion", "000000")]
-    with patch.object(pg_cleaner.image, "ImageStore") as mock_image_store:
-        pg_cleaner._clean_single_row(row, image_store_dict, "000000")
-
-    mock_image_store.assert_has_calls(expected_calls)
-
-
-def test_log_and_check_totals_raises_when_number_of_images_cleaned_is_wrong(
-    monkeypatch,
-):
-    monkeypatch.setattr(pg_cleaner.image.ImageStore, "total_items", 1)
-    expected_calls = [
-        call.info("Total images cleaned:  2"),
-        call.info("Image Totals breakdown:  {('abc', '000'): 1, ('def', '000'): 1}"),
-        call.warning("total_images_sum NOT EQUAL TO total_rows!"),
-        call.warning("total_images_sum: 2"),
-        call.warning("total_rows: 3"),
-    ]
-    image_store_dict = pg_cleaner.ImageStoreDict()
-    image_store_dict[("abc", "000")]
-    image_store_dict[("def", "000")]
-    with patch.object(pg_cleaner, "logger") as mock_logger:
-        with pytest.raises(AssertionError):
-            pg_cleaner._log_and_check_totals(3, image_store_dict)
-
-    assert mock_logger.has_calls(expected_calls)
-
-
-def test_log_and_check_totals_logs(monkeypatch):
-    monkeypatch.setattr(pg_cleaner.image.ImageStore, "total_items", 1)
-    expected_calls = [
-        call.info("Total images cleaned:  2"),
-        call.info("Image Totals breakdown:  {('abc', '000'): 1, ('def', '000'): 1}"),
-    ]
-    image_store_dict = pg_cleaner.ImageStoreDict()
-    image_store_dict[("abc", "000")]
-    image_store_dict[("def", "000")]
-    with patch.object(pg_cleaner, "logger") as mock_logger:
-        pg_cleaner._log_and_check_totals(2, image_store_dict)
-
-    assert mock_logger.has_calls(expected_calls)
+# def test_clean_prefix_loop_raises_with_long_prefix(monkeypatch):
```

Comment:
Can this file be deleted? Should these tests be updated?

Reply:
I think this file needs to be deleted, but I'm not sure if it should be done in this PR. From what I understand this was mostly used once to clean up the database, but I might not know enough to know not remove it. For context, [this is the PR adding pg_cleaner.](https://github.com/creativecommons/cccatalog/pull/517)